File: src/ServiceMessageModel.java
public class ServiceMessageModel {
    public static int SERVICE_PUBLISH_REQUEST = 101;
    public static int SERVICE_UNPUBLISH_REQUEST = 102;
    public static int SERVICE_DISCOVER_REQUEST = 201;
    public static int SERVICE_PUBLISH_OK = 103;
    public static int SERVICE_DISCOVER_NOT_FOUND = 202;
    public static int SERVICE_DISCOVER_OK = 203;

    int code;
    String data;

}

File: web_templates/js/ServiceMessageModel.js
class ServiceMessageModel {
    static SERVICE_PUBLISH_REQUEST = 101;
    static SERVICE_UNPUBLISH_REQUEST = 102;
    static SERVICE_PUBLISH_OK = 103;
    static SERVICE_PUBLISH_FAILED = 104;

    static SERVICE_DISCOVER_REQUEST = 201;
    static SERVICE_DISCOVER_NOT_FOUND = 202;
    static SERVICE_DISCOVER_OK = 203;

    constructor(code, data) {
        this.code = code;
        this.data = data;
    }
}

export default ServiceMessageModel;